chevah/github_hooks_server/server.py

```diff
@@ -114,32 +108,24 @@ def hook(request, hook_name):
     handle_event(event)
 
 
-@route('/buildmaster',  methods=['POST'])
-def buildmaster(request):
-    """
-    Buildbot integration entry point.
-    """
-    return BuildbotTryNotifier(CONFIGURATION)
```

Comment:
I removed `BuildbotTryNotifier` as well, since it wouldn't work after the Py3 upgrade, and I believe it is no longer used either.

Reply:
yes... buildmaster is no longer used.